kuma/javascript/src/article.jsx

```diff
@@ -191,6 +191,7 @@ function ArticleMetadata({ document }: DocumentProps) {
                 lastModifiedBy={document.lastModifiedBy}
                 lastModified={document.lastModified}
                 profileBaseURL={profileBaseURL}
+                documentLocale={document.locale}
```

Comment:
Note that the document locale does not always match the user's locale when viewing untranslated documents. If you want his date to be treated as part of the "chrome" rather part of the article, consider using l10n.getLocale() here instead.

Reply:
We should do that! 
We've already seen that if you view a Swedish page that hasn't been translate the "chrome" is in Swedish. E.g. "Senast uppdaterad:" instead of "Last updated:". So it doesn't make sense to say "Senast uppdaterad: Oct 12, 2019" since the swedes would have expected that to be "Senast uppdaterad: 12 okt. 2019"

Reply:
```
> date = new Date(Date.UTC(2019, 9, 12, 3, 0, 0));
Date Fri Oct 11 2019 23:00:00 GMT-0400 (Eastern Daylight Time)
> date.toLocaleString('sv-SE', {year: 'numeric', month: 'short', day: 'numeric'})
"11 okt. 2019"
```

Reply:
^ That's from the Web Console here in Firefox.

Reply:
Actually, how do you even get to pages where the "chrome" is in, say, Swedish but the content is still in English?
For example, the "Tools/Eyedropper" page hasn't been translated to Swedish but you can manually mess with the URL to become: https://developer.mozilla.org/sv-SE/docs/Tools/Eyedropper
But that doesn't exist as a link. Kumascript and stuff makes sure that if you're o https://developer.mozilla.org/sv-SE/docs/Tools the link to "Eyedropper" is only available in English. 

So for the rare few people who mess with the URL and trigger page renders where the chrome language != content language, I think these people can put up with the date formatting not being perfect.

Reply:
I.e. I'm not going to bother fixing it to use `l10n.getLocale()`.